C/CasADi/build_tarballs.jl

```diff
@@ -16,14 +16,18 @@ script = raw"""
 cd $WORKSPACE/srcdir/casadi
 install_license LICENSE.txt
 
+if [[ "$target" == *-mingw* ]]; then
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/windows.patch
+fi
+
 mkdir -p build
 cd build
 
 export CXXFLAGS="-fPIC -std=c++11"
 export CFLAGS="${CFLAGS} -fPIC"
 
 cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
-    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_clang.cmake \
```

Comment:
Do you mean something like this?

Reply:
I sometimes feel like you carry around a tome of incantations for all the weird things that can happen. 

It's amazing how much work has gone into all of this to make it work.